src/social_auth/backends/github.py

```diff
@@ -23,14 +23,14 @@
 
 
 # GitHub configuration
-GITHUB_AUTHORIZATION_URL = "https://github.com/login/oauth/authorize"
-GITHUB_ACCESS_TOKEN_URL = "https://github.com/login/oauth/access_token"
-GITHUB_USER_DATA_URL = "https://api.github.com/user"
+GITHUB_BASE_DOMAIN = getattr(settings, "GITHUB_BASE_DOMAIN", "github.com")
```

Comment:
@BYK when would `GITHUB_BASE_DOMAIN` change?

Reply:
When people are using self-hosted GitHub Enterprise

Reply:
are you sure this even works with GHE? i would probably avoid supporting something we historically haven't. folks can use normal SSO IMO.

Reply:
@BYK I see but we shouldn't we be handling that at a different level (like a database row)? Unless we are adding functionality that won't exist in the SaaS product.

Reply:
@dcramer @scefali - I mean I didn't really invent this. It was there for GitHub SSO and seems like these two things are sharing some code and config so I'm just trying to unify these.